released/packages/coq-list-string/coq-list-string.2.1.2/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+maintainer: "dev@clarus.me"
+homepage: "https://github.com/clarus/coq-list-string"
+dev-repo: "git+https://github.com/clarus/coq-list-string.git"
+bug-reports: "https://github.com/clarus/coq-list-string/issues"
+authors: ["Guillaume Claret"]
+license: "MIT"
+build: [
+  ["./configure.sh"]
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+remove: ["rm" "-R" "%{lib}%/coq/user-contrib/ListString"]
```

Comment:
Can you please delete the `remove` and `flags` lines (since file tracking for uninstall is now automatic).